src/API/Orders.php

```diff
@@ -107,15 +170,33 @@ protected function get_formatted_item_data( $object ) {
 			$data = $object->get_data();
 		}
 
+		$extra_fields      = false === $fields ? array() : array_intersect( $extra_fields, $fields );
 		$format_decimal    = array( 'discount_total', 'discount_tax', 'shipping_total', 'shipping_tax', 'shipping_total', 'shipping_tax', 'cart_tax', 'total', 'total_tax' );
 		$format_date       = array( 'date_created', 'date_modified', 'date_completed', 'date_paid' );
 		$format_line_items = array( 'line_items', 'tax_lines', 'shipping_lines', 'fee_lines', 'coupon_lines' );
 
+		// Add extra data as necessary.
+		$extra_data = array();
+		foreach ( $extra_fields as $field ) {
+			switch ( $field ) {
+				case 'customer':
+					$extra_data['customer'] = $this->get_customer_by_id( $data['customer_id'] );
+					break;
+				case 'products':
+					$extra_data['products'] = $this->get_products_by_order_id( $object->get_id() );
+					break;
+			}
```

Comment:
Add support for customer and products data, only retrieved if specifically set in `_fields: ...`